packages/wrapper/test/editorAppClassic.test.ts

```diff
@@ -49,6 +49,10 @@ describe('Test EditorAppClassic', () => {
         const app = new EditorAppClassic('test', createBaseConfig('classic'));
         expect(app.isAppConfigDifferent(orgConfig, config, false)).toBeFalsy();
 
+        if (!config.codeResources) {
+            config.codeResources = {};
+        }
```

Comment:
nitpicking, you can use `config.codeResources ??= {}`

btw, I've noticed you don't seem to use the `strict-boolean-expressions` eslint rule. It seems import to code quality and reliability, is there a reason for not using it?

Reply:
> nitpicking, you can use config.codeResources ??= {}

I didn't really use it like this before. Thanks for the hint.

> btw, I've noticed you don't seem to use the `strict-boolean-expressions` eslint rule. It seems import to code quality and reliability, is there a reason for not using it?

Uih, I just integrated that and I wasn't aware of the issues it revealed. The checking is taken from Langium. After seeing the effects of this rule I will recommend to enable it there as well.

Reply:
@CGNonofr btw, have you ported any configuration to eslint 9 already? It seems to be not very straight forward. I did not really spent time to investigate / understand it, though.

Reply:
No, eslint is the only package not up to date on the monaco-vscode-api side, I didn't investigate either, everything seems to broke, I will probably wait a little bit for all the plugin to update

Reply:
Done ⬇️